permissions/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
It would be good if you add some tests in the app.

Reply:
Yeah, I'm working on it.